index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="Your description goes here">
+    	<meta name="keywords" content="one, two, three">
+
+		<title>Blank Template</title>
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+		<!-- external CSS link -->
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+
+	</head>
+	<body>
+		<div id="container">
+			<div class="header">
+				
+				<h1> Relaxr </h1>
+
+				<ul class="nav">
+					<li><a href="">About</a></li>
+					<li><a href="">FAQ</a></li>
+					<li><a href="">Team</a></li>
+					<li><a href="">Contact Us</a></li>
+					<li><a href="">Blog</a></li>
+				</ul>
+			</div>
+
+			<div class="left">
+			
+				<div class="blog blog-1">
+					<div class="content-blogs">
+						<h2>How I implemented Relaxr in 2 weeks and changed my life.</h2>
+						<img src="images/blog_photo1.jpg">
```

Comment:
@symxing images are broken on your site because the paths are wrong, fix this and they'll work!